icons/radio-tower.svg

```diff
@@ -0,0 +1,19 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M4.9 16.1C1 12.2 1 5.8 4.9 1.9" />
+  <path d="M7.8 13.2c-2.3-2.3-2.3-6.1 0-8.5" />
+  <circle cx="12" cy="9" r="2" />
+  <path d="M16.2 4.8c2.3 2.3 2.3 6.1 0 8.5" />
+  <path d="M19.1 1.9C23 5.8 23 12.1 19.1 16" />
+  <line x1="7" x2="17" y1="20" y2="20" />
+  <polyline points="5,22 12,15 19,22" />
```

Comment:
Not sure about the tower being disconnected from the transmitter, how about something like this?

![image](https://user-images.githubusercontent.com/17746067/232312675-2bcce136-a40e-4675-98a3-4096dcbc78ae.png)

```suggestion
  <path d="M4.9 16.1C1 12.2 1 5.8 4.9 1.9" />
  <path d="M7.8 4.7a6.14 6.14 0 0 0-.8 7.5" />
  <path d="M12 11a2 2 0 1 0 0-4 2 2 0 0 0 0 4Z" />
  <path d="M16.2 4.8c2 2 2.26 5.11.8 7.47" />
  <path d="M19.1 1.9a9.96 9.96 0 0 1 0 14.1" />
  <path d="M9.5 18h5" />
  <path d="m8 22 4-11 4 11" />
```